shared/constants/mmi-controller.ts

```diff
@@ -3,10 +3,21 @@ import { TransactionMeta } from '@metamask/transaction-controller';
 import { TransactionUpdateController } from '@metamask-institutional/transaction-update';
 import { CustodyController } from '@metamask-institutional/custody-controller';
 import { SignatureController } from '@metamask/signature-controller';
-import { NetworkController } from '@metamask/network-controller';
+import {
+  NetworkController,
+  NetworkControllerGetStateAction,
+  NetworkControllerSetActiveNetworkAction,
+} from '@metamask/network-controller';
 // TODO: Remove restricted import
 // eslint-disable-next-line import/no-restricted-paths
```

Comment:
It seems like we can drop these comments since the order of the imports is now different:

``` suggestion
```